sources/engine/Stride.BepuPhysics/Stride.BepuPhysics.Navigation/Processors/RecastMeshSystem.cs

```diff
@@ -364,15 +373,15 @@ public static DtStatus FindFollowPath(DtNavMeshQuery navQuery, long startRef, lo
                 len = STEP_SIZE / len;
             }
 
-            RcVec3f moveTgt = RcVecUtils.Mad(iterPos, delta, len);
+            RcVec3f moveTgt = RcVec.Mad(iterPos, delta, len);
 
             // Move
-            navQuery.MoveAlongSurface(polys[0], iterPos, moveTgt, filter, out var result, ref visited);
+            navQuery.MoveAlongSurface(polys[0], iterPos, moveTgt, filter, out var result, visited, out nvisited, 16);
```

Comment:
16 should be replaced with visited.Length afaict ?

Reply:
absolutely!